src/components/Header.tsx

```diff
@@ -0,0 +1,446 @@
+                            className={`p-4 cursor-pointer ${
+                                activeMenu === "members" ? "text-red-600" : ""
+                            }`}
+                            onClick={() => handleMenuClick("members")}
+                        >
+                            {t('members')}
+                        </div>
+                        {activeMenu === "members" && (
+                            <div className="pl-4">
+                                <div
+                                    className="py-2 cursor-pointer hover:text-red-600"
+                                    onClick={() => {
+                                        if (accessToken) {
+                                            navigate("/account-info");
+                                        } else {
+                                            navigate("/login");
+                                        }
+                                    }}
+                                >
+                                    {t('cgv_account')}
+                                </div>
+                                <div className="py-2 cursor-pointer hover:text-red-600">
+                                    {t('benefits')}
+                                </div>
+                            </div>
+                        )}
+
+                        <div
+                            className={`p-4 cursor-pointer ${
+                                activeMenu === "cultureplex" ? "text-red-600" : ""
+                            }`}
+                            onClick={() => handleMenuClick("cultureplex")}
+                        >
+                            {t('cultureplex')}
+                        </div>
+                        {activeMenu === "cultureplex" && (
+                            <div className="pl-4">
+                                <div className="py-2 cursor-pointer hover:text-red-600">
+                                    {t('online_store')}
+                                </div>
+                                <div className="py-2 cursor-pointer hover:text-red-600">
+                                    {t('rent_and_group')}
+                                </div>
+                                <div className="py-2 cursor-pointer hover:text-red-600">
+                                    {t('e_cgv')}
+                                </div>
+                                <div className="py-2 cursor-pointer hover:text-red-600">
+                                    {t('egift')}
+                                </div>
+                                <div className="py-2 cursor-pointer hover:text-red-600">
+                                    {t('cgv_rules')}
+                                </div>
+                            </div>
+                        )}
+                    </div>
+                )}
+            </div>
+        </HeaderContainer>
+    );
+};
+
+const HeaderContainer = styled.div`
```

Comment:
E đưa những css này ra file riêng nha, đặt trong public
Ngoài ra e có thể reponsive bằng tailwind nha
Responsive Prefixes
sm: (640px and up)
md: (768px and up)
lg: (1024px and up)
xl: (1280px and up)
2xl: (1536px and up)
NHững chỗ này dùng tailwind được, e dùng tailwind luôn